workflows/create-pull-request.yaml

```
@@ -0,0 +1,59 @@
+name: Create Pull request
+
+on:
+  schedule:
+    # Runs "at minute 44 past midnight" (see https://crontab.guru)
+    - cron: "44 0 * * *"
+
+jobs:
+  createPullRequest:
+    runs-on: ubuntu-latest
+    env:
+      bot_username: BotName
+      bot_email: BotMail@gmail.com
+    strategy:
+      fail-fast: false
+      matrix:
+        repos: ["de-german", "fr-french"]
+    steps:
+      - name: Fetch translation repository
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.PAT }}
+          repository: solidity-docs-test/${{ matrix.repos }}
+          fetch-depth: "0"
+
+      - name: Fetch this repository
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.PAT }}
+          repository: solidity-docs/translation-guide
+          path: .github-workflow
+
+      - name: Create merge commit
+        id: merge
+        run: |
+          ./.github-workflow/scripts/merge-conflicts.sh
+          ./.github-workflow/scripts/generate-PR-body.sh
+
+      - name: Remove this repository
+        run: |
+          rm -rf .github-workflow
+
+      - name: Create Pull Request
+        id: cpr
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: "${{ secrets.PAT }}"
+          commit-message: "Update report"
```

I'd put the PR title here. Otherwise there will be lots of same-looking `Update report` commits in repo history :)
```suggestion
          commit-message: "${{ env.pr_title }}"
```

---

This is good idea. By the way, the commit message in ReactJS is "merging all conflicts" https://github.com/reactjs/hu.reactjs.org/pull/94/commits

---

done